dynamic/Dockerfile

```diff
@@ -1,29 +1,37 @@
-FROM ubuntu:21.04
+# syntax = docker/dockerfile:1.2
```

Comment:
This comment can be removed?

Reply:
@mdonkers That's a directive for BuildKit: https://docs.docker.com/develop/develop-images/build_enhancements/#overriding-default-frontends